lib/stronger_parameters/constraints/nil_string_contraint.rb

```diff
@@ -0,0 +1,15 @@
+require 'stronger_parameters/constraints'
+
+module StrongerParameters
+  class NilStringConstraint < Constraint
+    NULL_VALUES = [nil, '', 'undefined', 'UNDEFINED']
```

Comment:
found a bunch of 'undefined' but no 'UNDEFINED' so should be good ... unless someone knows of any usecase I'll just remove it ...